api/v1beta1/nodepool_types.go

```diff
@@ -705,6 +706,12 @@ type KubevirtNodePoolPlatform struct {
 	// +optional
 	// +kubebuilder:validation:Enum=Enable;Disable
 	NetworkInterfaceMultiQueue *MultiQueueSetting `json:"networkInterfaceMultiqueue,omitempty"`
+
+	// NetworkNames specify the networks attached to the nodes, they can be special value "default" with means pod network
+	// or value with format "[namespace]/[name]" to reference at multus network attachment definition
+	//
+	// +optional
+	NetworkNames []string `json:"networkNames"`
```

Comment:
Thinking towards the future, maybe we need this to be a struct. something like...

```
Networks []KubevirtPlatformNetwork `json:"networks,omitempty"`
.
.
.
type KubevirtPlatformNetworks struct {
   // required
    Name string `json:"name"`
}
```

that way we can add more network specific tunings in the future. for example, someone might want to define their network interface's model or other tunings in the future, which we would need to expose in the nodepool api.

Reply:
Let's add some more so we can design here how to pass those too and we are prepare for the future.